python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -139,7 +204,15 @@ def execute_queries(
 
         return results if fetch_results else None
 
-    def load_table_from_local_parquet(self, src, table):
+    @public
+    def load_table_from_local_parquet(self, src: str, table: str):
```

Comment:
@gibsondan git blame says you were the last one to work on this fn, can you confirm that the doc string is accurate?

Reply:
This looks right to me